tests/director/api/application_templates_api_test.go

```diff
@@ -194,3 +195,63 @@ func TestRegisterApplicationFromTemplate(t *testing.T) {
 	require.Equal(t, "test new-value", *outputApp.Application.Description)
 	saveExample(t, createAppFromTmplRequest.Query(), "register application from template")
 }
+
+func TestUpdateApplicationTemplateWebhooks(t *testing.T) {
```

Comment:
Its more like TestAddWebhookToApplicationTemplate - as you are using addWebhook mutation and not updateApplicationTemplate here